client/influxdb.go

```diff
@@ -642,6 +646,19 @@ func (c *Client) Addr() string {
 	return c.url.String()
 }
 
+// checkPointTypes ensures no unsupported types are submitted to influxdb, returning error if they are found.
+func checkPointTypes(p Point) error {
+	for _, v := range p.Fields {
+		switch v.(type) {
+		case int, int8, int16, int32, int64, uint, uint8, uint16, uint32, float32, float64, bool, []byte, string, nil:
```

Comment:
Oops.  Sorry... missed an important word in my comment.  Yes, `[]byte` is _not_ supported via the line protocol. :)